docs/05-technical-reference/svls-03-switching-registries.md

```diff
@@ -57,4 +57,4 @@ This implementation has a fallback mechanism that works as follows:
 1. Every 5 minutes, Function Controller checks if there is the `serverless-registry-config` Secret CR in the Namespace with your Function specifying alternative registry to push the Function's image to.
 2. If it doesn't find such a Secret CR, Function Controller uses the credentials to the default registry specified in the `serverless-registry-config-default` Secret CR.
 
-This mechanism also leaves room for a lot of flexibility as you can easily switch between external registries or move back to the internal one. If you remove the `serverless-registry-config` Secret CR or update it with credentials to a different external registry, you don't lose any images. Function Controller detects any changes in the Secret CR and the images are rebuilt automatically, using cache and delta updates. If you modify the username and password to the registry, the [admission webhook](./svls-07-supported-webhooks.md#admission-webhook) automatically encodes these credentials to base64 and sets them as a value under the `.dockerconfigjson` entry in the Secret CR. These credentials later serve Kubernetes to pull images of deployed Function from the registry, and allow Kaniko to push any newly built or rebuilt images to this registry.
+This mechanism also leaves room for a lot of flexibility as you can easily switch between external registries or move back to the internal one. If you remove the `serverless-registry-config` Secret CR or update it with credentials to a different external registry, you don't lose any images. Function Controller detects any changes in the Secret CR and the images are rebuilt automatically, using cache and delta updates. If you modify the username and password to the registry, the admission webhook automatically encodes these credentials to base64 and sets them as a value under the `.dockerconfigjson` entry in the Secret CR. These credentials later serve Kubernetes to pull images of deployed Function from the registry, and allow Kaniko to push any newly built or rebuilt images to this registry.
```

Comment:
Why do we remove the supported webhooks doc since they're still used?

Reply:
Webhooks still are used. Now we removed outdated documentation with not important details but this description still is true.

Reply:
As discussed, please prepare a document that briefly describes the used webhooks.

Reply:
we are removing the documentation about webhooks because it became very outdated and clearly noone was looking into it ( noone reported any bugs ).
Users may know about webhooks.. in general.
But they dont really configure them, or influence their behaviour in any way.
Keeping documentation about internal mechanics of the module is a lot of burden with minumum value.